images/base/files/usr/local/bin/entrypoint

```diff
@@ -378,6 +383,22 @@ select_iptables() {
   update-alternatives --set ip6tables "/usr/sbin/ip6tables-${mode}" > /dev/null
 }
 
+fix_certificate() {
```

Comment:
we are calling this twice if both ips, the ipv4 and ipv6 has changed, is that the expected?

another question, is this keeping the localhost address, that we added, on the certificate?

Reply:
> we are calling this twice if both ips, the ipv4 and ipv6 has changed, is that the expected?

good point, we should store if either changed and then run it once at the end.

> another question, is this keeping the localhost address, that we added, on the certificate?

we use the same kubeadm config (`kind/kubeadm.conf`) so it should be

Reply:
thanks for pointing it out, updated to regenerate only once